server/src/main/java/org/apache/iotdb/db/qp/physical/sys/StorageGroupMNodePlan.java

```diff
@@ -30,19 +30,27 @@
 import java.util.Objects;
 
 public class StorageGroupMNodePlan extends MNodePlan {
+
   private long dataTTL;
 
   private int alignedTimeseriesIndex;
 
+  private long majorVersion;
+
+  private long minorVersion;
+
   public StorageGroupMNodePlan() {
     super(false, Operator.OperatorType.STORAGE_GROUP_MNODE);
   }
 
-  public StorageGroupMNodePlan(String name, long dataTTL, int childSize) {
```

Comment:
The original interface is retained. If the majorversion and minorversion values are not setted, the default values can be used.

Reply:
Sure, thanks